lib/iris/plot.py

```diff
@@ -351,7 +351,12 @@ def _check_bounds_contiguity_and_mask(coord, data, atol=None, rtol=None):
             )
 
             not_masked_at_discontiguity_along_y = np.any(
-                np.logical_and(mask_invert[:-1,], diffs_along_y)
+                np.logical_and(
+                    mask_invert[
+                        :-1,
+                    ],
+                    diffs_along_y,
+                )
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?